apollo-gradle-plugin-external/gradle.properties

```diff
@@ -2,3 +2,4 @@ POM_ARTIFACT_ID=apollo-gradle-plugin-external
 POM_NAME=Apollo Gradle Plugin (not shadowed)
 POM_DESCRIPTION=Gradle plugin for generating java/kotlin classes for graphql files
 
+kotlin.stdlib.default.dependency=false
```

Comment:
Could that be set once in the rootProject? I think Gradle properties have a notion of "hierarchy"

Reply:
You're right! TIL